.github/workflows/ci.yml

```diff
@@ -7,6 +7,7 @@ on:
       - develop
       - feat/facilities
       - feat/products
+      - chore/ui_cleanup
```

Comment:
@artursudnik We should enable the CI on all PR's which has a target of `develop` or `master`